corgi/tasks/common.py

```diff
@@ -57,3 +59,19 @@ def get_last_success_for_task(task_name):
     return (
         last_success - timedelta(minutes=30) if last_success else timezone.now() - timedelta(days=3)
     )
+
+
+def _create_relations(build_ids, external_system_id, product_ref, relation_type) -> int:
+    no_of_relations = 0
+    for build_id in build_ids:
+        if not build_id:
+            continue
```

Comment:
Not clear on what this is doing - won't we always have a build_id?

Reply:
I found that in some cases the collectors.pulp.Pulp().get_module_data or collectors.brew.Brew.persist_modules() generators where returning None values, so this is to a void trying to persist those as relations which causes a DB Error.

Reply:
Noting that here might be good - thanks for explaining.